### ISSUE TITLE
grpc++.h not installed, should it be?

### ISSUE THREAD

**Issue by emmenlau** [grpc/grpc#3416]
I tried to compile the newest greeter_client.cc with grpc 0_11_0 release, but it fails, due to a missing header. I can see the header in the sources, but its not installed:
```
greeter_client.cc:38:27: fatal error: grpc++/grpc++.h: No such file or directory

**Comment by ctiller** [comment]
It should be. It's corrected in an upcoming 0.11.1 release (and in the 0.11 branch).

**Comment by ecsgz** [comment]
Hi, I have the same problem when trying to compile the sample cpp files in Ubuntu 14.04 and within flatbuffers-1.7.1. All tests were passed successfully with `./flattests`, but there is at all no grpc++.h file!

Please can you help?
